core/helpers_test.go

```diff
@@ -1,11 +1,11 @@
 package core
 
 import (
-	"testing"
 	"reflect"
+	"testing"
 )
 
-func checkErrors(t *testing.T, name string, index int, err error, expected error) {
+func checkErrors(t *testing.T, name string, index int, err Errs, expected Errs) {
```

Comment:
I simply thought, that as we were using our own type, it would make more sense to use it there as well. But that would indeed work with both, I'll revert it.